UOP1_Project/Assets/Dialogue/conversations/nice tree.asset

```diff
@@ -23,4 +23,4 @@ MonoBehaviour:
     fontstyle: 3
     Speaking: {fileID: 10907, guid: 0000000000000000f000000000000000, type: 0}
     NameOveride: character2
-  triggered_once: 1
+  triggered_once: 0
```

Comment:
There might be problems here, remind me about this later on, @dkaloger

Reply:
no it was done very intentionally . dialogs are ment to only be triggered once even after exiting and the rentering playmode

Reply:
Ok, then it's all good. Just wanted to make sure that the persistence is intended